app/logic/solana_logic.rb

```diff
@@ -490,6 +490,8 @@ def cli_request(cli_method, rpc_urls)
         Rails.logger.error "CLI TIMEOUT\n#{e.class}\nRPC URL: #{rpc_url}"
         ''
       end
+      # puts "cli request #{cli_method.split(' ')[0]}\n #{response_json}"
```

Comment:
Do we want this comment?

Reply:
I would leave it for now, 'cause it's useful for debugging